lib/msal-browser/src/app/ClientApplication.ts

```diff
@@ -528,7 +528,11 @@ export abstract class ClientApplication {
         const atbcMeasurement = this.performanceClient.startMeasurement(PerformanceEvents.AcquireTokenByCode, request.correlationId);
 
         try {
-            if (request.code) {
+            if (request.code && request.nativeAccountId) {
```

Comment:
Optional. You could replace it with:
```
if (request.code) {
  if (request.nativeAccountId) {
    throw AuthError.createUnexpectedError("Request cannot contain both SPA code and native account id.");
  }
  // rest of the code
  ...
}
```

Reply:
I feel the current if-else-if structure is more readable.. but thanks for the suggestion!